mlflow/pyfunc/__init__.py

```diff
@@ -628,6 +633,35 @@ def __repr__(self):
         return yaml.safe_dump({"mlflow.pyfunc.loaded_model": info}, default_flow_style=False)
 
 
+def check_requirements_and_local_installed_mismatch(local_path):
+    req_file_path = os.path.join(local_path, "requirements.txt")
```

Comment:
```suggestion
    req_file_path = os.path.join(local_path, _REQUIREMENTS_FILE_NAME)
```

Can we de-hardcode this?